src/Microsoft.TemplateEngine.TemplateLocalizer.Core/TemplateStringUpdater.cs

```diff
@@ -0,0 +1,129 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text.Json;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.TemplateEngine.TemplateLocalizer.Core
+{
+    internal static class TemplateStringUpdater
+    {
+        /// <summary>
+        /// Updates the templatestrings.json files for given languages with the provided strings.
+        /// </summary>
+        /// <param name="strings">Template strings to be included in the templatestrings.json files.</param>
+        /// <param name="templateJsonLanguage">The language of the <paramref name="strings"/>.</param>
```

Comment:
Can you mention that `strings`/`templateJsonLanguage` come from/is for `template.json file`?